$OpenBSD: patch-deps_eleveldb_rebar_config,v 1.1.1.1 2012/08/24 07:26:29 jmatthew Exp $

Use external leveldb rather than trying to patch the bundled
tarball to get it to build.

--- deps/eleveldb/rebar.config.orig	Thu Aug  2 15:29:21 2012
+++ deps/eleveldb/rebar.config	Wed Aug  8 11:23:46 2012
@@ -6,11 +6,11 @@
              %% Make sure to set -fPIC when compiling leveldb
              {"CFLAGS", "$CFLAGS -fPIC"},
              {"CXXFLAGS", "$CXXFLAGS -fPIC"},
-             {"DRV_CFLAGS", "$DRV_CFLAGS -Werror -I c_src/leveldb/include"},
-             {"DRV_LDFLAGS", "$DRV_LDFLAGS c_src/leveldb/libleveldb.a c_src/system/lib/libsnappy.a -lstdc++"}
+             {"DRV_CFLAGS", "$DRV_CFLAGS -Werror -I /usr/local/include"},
+             {"DRV_LDFLAGS", "$DRV_LDFLAGS -L /usr/local/lib -lleveldb -lstdc++"}
              ]}.
 
-{pre_hooks, [{'get-deps', "c_src/build_deps.sh get-deps"},
-             {compile, "c_src/build_deps.sh"}]}.
+%% {pre_hooks, [{'get-deps', "c_src/build_deps.sh get-deps"},
+%%              {compile, "c_src/build_deps.sh"}]}.
 
-{post_hooks, [{clean, "c_src/build_deps.sh clean"}]}.
+%% {post_hooks, [{clean, "c_src/build_deps.sh clean"}]}.
